src/css/style.css

```diff
@@ -114,6 +134,9 @@ body {
   border: 3px solid var(--gray);
   background-color: var(--button-dark-blue);
 }
+.btn-sm {
+  width: 30%;
```

Comment:
@Katsu08 please check this one here, not sure if it works in several sizes on desktop

Reply:
Doesn't work in very small screens.

![imagen](https://user-images.githubusercontent.com/66505715/157894456-f21ce2b5-0415-408a-89a3-3ef3d75da29e.png)
![imagen](https://user-images.githubusercontent.com/66505715/157894550-61d802a6-06d1-4b34-ae5a-cec034924c06.png)

Reply:
Please add min-width.
What do you think about to use: `min-width: min-content` ? 

![imagen](https://user-images.githubusercontent.com/66505715/157894596-51e3cc29-2983-49a1-aaa1-5f545c984eb0.png)